ntbs-service/Pages/Alerts/TransferRequest.cshtml.cs

```diff
@@ -101,6 +109,21 @@ public async Task<IActionResult> OnPostAsync()
             return RedirectToPage("/Notifications/Overview", new { NotificationId });
         }
 
+        private async Task SetValuesForValidation()
+        {
+            await GetRelatedEntities();
+            ValidationService.TrySetFormattedDate(TransferRequest, "TransferRequest", nameof(TransferRequest.TransferDate), FormattedTransferDate);
+            SetDatesOnRequestForValidation(TransferRequest);
+        }
+
+        private void SetDatesOnRequestForValidation(TransferViewModel transferRequest)
+        {
+            transferRequest.NotificationStartDate =
+                Notification.ClinicalDetails.StartingDate ?? Notification.NotificationDate;
+            transferRequest.LatestTransferDate = Notification.TreatmentEvents
+                .FirstOrDefault(te => te.TreatmentEventType == TreatmentEventType.TransferIn)?.EventDate;

```

Comment:
Are the notification's treatment events guaranteed to be ordered reverse chronologically? I would have expected you to need an ordering here